docs/plugins.md

```diff
@@ -1548,6 +1548,17 @@ These plugins provide new markers or news ways of converting abstract data into
 			<a href="https://github.com/yakitoritabetai">Kenta Hakoishi</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/triedeti/Leaflet.streetlabels">Leaflet.streetlabels</a>
+		</td>
+		<td>
+			A Leaflet plugin to show labels following the paths of polylines. An extension of yakitoritabetai Leaflet.LabelTextCollision(<a href="https://triedeti.github.io/Leaflet.streetlabels/">demo</a>).
```

Comment:
Nitpick: missing space in `Leaflet.LabelTextCollision(`

Reply:
@mourner Done! Thanks for the fast review, and keep up the good work!